script/openqa-bootstrap

```diff
@@ -92,7 +95,8 @@ fi
 if zypper -n search -x qemu-hw-display-virtio-gpu-pci; then
     pkgs+=(qemu-hw-display-virtio-gpu qemu-hw-display-virtio-gpu-pci)
 fi
-zypper -n install --no-recommends "${pkgs[@]}"
+packages="${pkgs[@]}"
+retry -e -s 30 -r 7 -- sh -c "zypper -n --gpg-auto-import-keys ref && zypper -n in--no-recommends $packages"
```

Comment:
With that change we would do

```
zypper ref
zypper in retry
retry zypper ref
retry zypper in
```

which is quite inefficient in the happy path when retry is not needed. I suggest the following

```
command -v retry >/dev/null || zypper --n --gpg-auto-import-keys in retry
retry -e -s 30 -r 7 -- sh -c "zypper -n --gpg-auto-import-keys ref && zypper -n in--no-recommends $packages"
```

this way in the best case retry already exists and we only call `zypper ref`. In case retry needs to be installed the initial install request for `retry` would refresh as desired but at this point we don't need the additional repositories we added so we do not necessarily need a `ref`.